src/app/components/harvest/pages/list/list.component.spec.ts

```diff
@@ -150,4 +155,29 @@ describe("ListComponent", () => {
     flush();
   }));
 
+  it("should show created dates in the users local dateTime", () => {
+    const mockUserTimeZone = "Australia/Perth"; // +08:00 UTC
+    const harvestUTCCreatedAt = DateTime.fromISO("2020-01-01T00:00:00.000Z");
+    const expectedLocalCreatedAt = "2020-01-01 08:00:00";
+    defaultHarvest = new Harvest(generateHarvest({ createdAt: harvestUTCCreatedAt }));
+
+    // To simplify tests, set the Luxon.Settings.defaultZone to a mock timezone (+08:00 UTC)
+    Settings.defaultZone = mockUserTimeZone;
```

Comment:
is this reset after the test?

Reply:
Thanks!

I was sure to explicitly reset the global Luxon state in the `afterEach()`

![image](https://user-images.githubusercontent.com/33742269/223029942-3ff4a823-290b-4ad8-8a4a-ff0edc636bbb.png)

[here](https://github.com/QutEcoacoustics/workbench-client/pull/2029/files#diff-684d91b066ab324d51e4159d8ab8c266486c3d2800a71ea15a4def565dcfe6caR109)